src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  gap: $dayBoxGapSize;
+  width: $calendarWidth;
+  padding: 20px $dayBoxPadding;
+
+  @each $day, $number in $weekDays {
```

Comment:
All of your loops i suggest dat u can add them in another scss 'like loops.scss' put into _utiles folder, and import them in your main)

Reply:
agree